### PR TITLE
Milestone Four - Logic

### PR DESCRIPTION
**Pull request by geraldgsh**
1. Created a new logic folder inside src
2. Initialize two javascript files there: calculate.js and operate.js.
3. Added the big.js package to project dependencies.
4. Created default method: calculate.
5. Added calculator data object (next, total) and a button name (buttonName) as arguments.
6. Added three properties: total, next and operation as calculator data object.
7. Added buttonName == "+/-" tototal and next by -1 to make it negative or positive.
8. Added 'A/C' to clear next, total and operation
9. Added '%' as modulo to multiply total by 0.01
10. Addded '=' to initiate calculations
11. Added a condition if buttonName includes one of the mathematically operands to initialize operate module
12. Imported the big.js module from the corresponding npm package.
13. Implemented one default function named operate.
14. Added numberOne, numberTwo and operation as arguments.
15. Setup value of the operation argument that perform a simple math operation. For instance if operation == "-", it should return the result of subtracting the two passed numbers.
16. Wrap the numbers with big.js classes to operate on big numbers.
17. Add condition If the buttonName in calculate.js is equal to any operation (e.g. +, -, x, ÷, %), operate.js should be used to calculate the total property.

### CODE REVIEW

**Comment by geraldgsh** [comment]
NOTE to TSE: 

I've mistakenly merge milestone 4 before this pull request

File: .eslintrc.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
   "extends": ["airbnb", "plugin:react/recommended"],
   "plugins": ["react"],
   "rules": {
-    "linebreak-style": "off",
     "react/prop-types": "off",
```
Comment (bellom) [inline .eslintrc.json]: You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @geraldgsh :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by geraldgsh** [comment]
Noted @bellom and congratulation on completing the Mircoverse program

**Comment by bellom** [comment]
@geraldgsh Thank you, see you at the top soon.

seems you forgot to remove this 🙂 `"react/prop-types": "off",`
check out React linter config file -> https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

**Review by bellom — APPROVED** [review]
## Hi @geraldgsh  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**